base/Web/Kaelin.kind

```diff
@@ -40,24 +40,45 @@ Web.Kaelin: App(Web.Kaelin.State)
           switch U16.eql(event.code) {
             'D':
               case state {
-                init: []
-                void: []
                 game:
-                  // TODO: Web.Kaelin.Map.swap(i, ca, cb, map)
-                  // let old_pos = state.pos
-                  // let new_pos = case old_pos {
-                  //   new: Web.Kaelin.Coord.new(U32.add(old_pos.i,1u),old_pos.j)
-                  // }
-                  // let {map,ent} = Web.Kaelin.Map.pop(old_pos, state.map)
-                  // let map = case ent {
-                  //   none: map
-                  //   some: Web.Kaelin.Map.push(new_pos, ent.value, map)
-                  // }
-                  //let state = Web.Kaelin.State.game(state.room, state.tick, new_pos, map)
+                  let state = Web.Kaelin.Action.move(Int.new(1,0), Int.new(0,0), state)
                   [App.Action.state!(state)]
-              }
+              } default []
+            'A':
+              case state {
+                game:
+                  let state = Web.Kaelin.Action.move(Int.new(0,1), Int.new(0,0), state)
+                  [App.Action.state!(state)]
+              } default []
```

Comment:
Couldn't this reappearing pattern be a bit more sugared? I think I've seen something for *case with only one statement* in the doc. Also I'm thinking about lexical "importing" (but not shadowing - in that case compiler would error out because of ambiguity) e.g. `Web.Kaelin.Action` into the scope somewhere before `switch U16.eql(event.code) {` allowing one to just write `move( Int.new( ... ) ... )` instead of `Web.Kaelin.Action.move( Int.new( ... ) ... )`.

@MaiaVictor thoughts?